ground/gcs/src/app/main.cpp

```diff
@@ -294,9 +300,13 @@ int main(int argc, char **argv)
 
     RunGuard &guard = RunGuard::instance(QStringLiteral(GCS_PROJECT_BRANDING)
                                          + QStringLiteral("_gcs_run_guard"));
+
     if (!guard.tryToRun()) {
         std::cerr << "GCS already running!" << std::endl;
-        return 1;
+
+        if (!parser.isSet(multipleInstancesOption)) {
+            return 1;
+        }
     }
```

Comment:
What I had done (but not pushed) was just wrap both crash reporter and runguard in the same `if (parser.values(doTestsOption).isEmpty()) {`. This is fine too.

Reply:
Sorry that I conflicted.  Might be other reasons to want multi.